### ISSUE TITLE
Add Antoine's eraser widget as a demo to the showcase

### ISSUE THREAD

**Issue by stephanrauh** [stephanrauh/ngx-extended-pdf-viewer#2505]
Antoine Legrain has implemented a nice component for erasing drawings. He also kindly shared it with us. Let's try to add this to the showcase!

<img width="973" alt="image" src="https://github.com/user-attachments/assets/34709cc8-565c-4204-9782-2a97997368b1">

[pdf-viewer.component.zip](https://github.com/user-attachments/files/16620471/pdf-viewer.component.zip)

_Originally posted by @legraina in https://github.com/stephanrauh/ngx-extended-pdf-viewer/discussions/2480#discussioncomment-10342908_

**Comment by legraina** [comment]
Thanks. I have a question about this eraser.
I'm removing all ink annotations and then adding back the ones that have been modified.
It seems that some annotations are not rendered (it's random). I suspect a timing issue, and I was wondering if I was waiting for the right event to add them back. For the moment, at initialization, I'm waiting for the first rendered event, and after erasing, I remove all ink annotations and then add the modified annotations immediately. Any idea of the cause ?

**Comment by stephanrauh** [comment]
Not really. I have to admit I'm a bit overwhelmed by the popularity of the editor features. I haven't spent much time yet reading and understanding the implementation, but you guys make me feel I should do that soon!

**Comment by stephanrauh** [comment]
I still think that this is a good idea, but I have to admit I barely cope with the flood of new tickets. In other words, I reluctantly close this ticket now. Sigh!
